Bugzilla/DB.pm

```diff
@@ -350,6 +350,11 @@ sub import {
     $Exporter::ExportLevel-- if $is_exporter;
 }
 
+sub sql_prefix_match {
+    my ($self, $column, $str) = @_;
+    return "$column LIKE " . $self->quote("$str%");
```

Comment:
this won't escape `LIKE` wildcards in `$str`: `\`, `%`, and `_`.
`_` in particular as there's nicks that contain underscores.

note escaping is different between mysql and sqlite (and possibly other dbs).
here's what i do for sqlite (where the calling code uses placeholders):
```
sub like_value {
    my ($field, $value) = @_;
    my $condition = $field . ' LIKE ?';
    if ($value =~ s/(?=[\\%_])/\\/g) {
        $condition .= ' ESCAPE \'\\\'';
    }
    return ($condition, '%' . $value . '%');
}
```